include/klee/Internal/Module/KInstruction.h

```diff
@@ -44,7 +44,8 @@ namespace klee {
 
   public:
     virtual ~KInstruction();
-    void printFileLine(llvm::raw_ostream &);
+    void printFileLine(llvm::raw_ostream &) const;
+    std::string printFileLine() const;
```

Comment:
Scratch that. `getFileLine()` isn't a good name because it sounds like it should return the line number. Perhaps `getLocation()`?